test/compiler/inline.jl

```diff
@@ -1526,3 +1526,16 @@ function oc_capture_oc(z)
     return oc2(z)
 end
 @test fully_eliminated(oc_capture_oc, (Int,))
+
+@testset "Inlining with unmatched type parameters" begin
+    # inlining of `apply_type`
+    let f(x) = Val{x}()
+        g() = [ Base.donotdelete(Val{i}()) for i in 1:10000 ]
+        srcs = (code_typed1(f, (Any,)),
+                code_typed1(g))
```

Comment:
```suggestion
                code_typed1(g, (Any,)))
```

Reply:
This resulted in a test error, I think the initial way of calling it was correct